README.md

```diff
@@ -116,6 +116,17 @@ After setup, it is good to check logs in all related modules for errors. Data im
   * "_srm.kafkacache.cleanup.interval.ms_": 3600000
   * "_srm.kafkacache.expiration.time.hours_": 3
 
+There are another important properties - `number of partitions` for topics `DI_COMPLETE`, `DI_ERROR`, `DI_SRS_MARC_BIB_RECORD_CREATED`, 
```

Comment:
DI_SRS_MARC_BIB_RECORD_CREATED and DI_EDIFACT_RECORD_CREATED are no longer in use. There is also topic related to parsing the MARC_Bib for Order creation

Reply:
done